pkg/codegen/python/gen.go

```diff
@@ -2524,32 +2524,41 @@ func (mod *modContext) genPropDocstring(w io.Writer, name string, prop *schema.P
 	}
 }
 
-func (mod *modContext) typeString(t schema.Type, input, acceptMapping bool, forDict bool) string {
+type typeStringOpts struct {
```

Comment:
Love this. If you happen to know how the various options affect the output, I think that'd be awesome to include in the field comments, or e.g. examples of what is an input type vs what isn't etc., but that's just me being greedy. Looks good either way!